test/models/work_test.rb

```diff
@@ -0,0 +1,95 @@
+require "test_helper"
+
+describe Work do
+  describe "validations" do
+    work = Work.first
```

Comment:
This should be a let or inside the `it` block

```suggestion
    let(:work) { Work.first }
```